examples/integrated-flows-examples/src/verification/app.ts

```diff
@@ -41,21 +41,21 @@ dotenv.config();
       case VerificationMethods.SMS:
         startSmsVerificationFlow(answers.phoneNumber);
         break;
-      case VerificationMethods.CALLOUT:
-        startCalloutVerificationFlow(answers.phoneNumber);
+      case VerificationMethods.PHONE_CALL:
```

Comment:
If we are moving to support smooth transition with both `CALLOUT` and `PHONE_CALL`; well need to duplicate the `case ` here:
```
case VerificationMethods.CALLOUT
case VerificationMethods.PHONE_CALL:
  ...
```